<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>136</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QR Reader</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>-10</y>
      <width>1500</width>
      <height>111</height>
     </rect>
    </property>
    <widget class="QPushButton" name="pushButton_okut">
     <property name="geometry">
      <rect>
       <x>148</x>
       <y>81</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Read QR Code</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_yaz">
     <property name="geometry">
      <rect>
       <x>776</x>
       <y>82</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Create QR Code</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_okut">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>500</width>
       <height>41</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string>Make sure that you are not blocking the Qr Code with the program then insert the QR Code in to your screen then press the button.</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_yaz">
     <property name="geometry">
      <rect>
       <x>658</x>
       <y>20</y>
       <width>371</width>
       <height>41</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string>               Type the value that you want to insert in QR Code.</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>920</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
